h2o-py/h2o/estimators/estimator_base.py

```diff
@@ -17,6 +17,7 @@
 from h2o.utils.compatibility import *  # NOQA
 from h2o.utils.shared_utils import quoted
 from h2o.utils.typechecks import assert_is_type, is_type, numeric, FunctionType
+from h2o.segment_models import H2OSegmentModels
```

Comment:
maybe this could be moved to `model` module, or keep it in `estimators` module if it's going to be used only here.

Reply:
moved, looks better next to the other import

thanks for the suggestion